docs(cli): document how to disable built-in providers (#8702)
Kilo-Org/kilocode: packages/opencode/src/kilocode/skills/kilo-config.md

}
```

### Disabling Built-in Providers

> Use `disabled_providers` to prevent specific providers from loading. This is useful when you want to exclude providers that are auto-detected via environment variables or that you don't want available in the model picker.

Author: ```suggestion
Use `disabled_providers` to prevent specific providers from loading. This is useful when you want to exclude providers that are built-in, or auto-detected via environment variables, from appearing in the model picker.
```

> 

Author: ```suggestion

For example, this configuration will hide all models from the built-in Kilo Gateway as well as any from the OpenAI provider which may be enabled automatically through environment variables.

```

```jsonc
{
  "$schema": "https://app.kilo.ai/config.json",
  "disabled_providers": ["kilo", "openai"],
}
```

The provider ID is the lowercase name used in the `provider/model` format (e.g., `kilo`, `openai`, `anthropic`, `google`, `groq`).

**Common provider IDs:**

| Provider  | ID          | Notes                   |
| --------- | ----------- | ----------------------- |
| Kilo      | `kilo`      | Default Kilo AI Gateway |
| OpenAI    | `openai`    |                         |
| Anthropic | `anthropic` |                         |
| Google    | `google`    |                         |
| Groq      | `groq`      |                         |
| Ollama    | `ollama`    | Local models            |
| LM Studio | `lmstudio`  | Local models            |

> 

Author: ```suggestion
```
